specs/processes.table

```diff
@@ -33,8 +33,10 @@ schema([
 ])
 extended_schema(WINDOWS, [
     Column("is_elevated_token", INTEGER, "Process uses elevated token yes=1, no=0"),
+    Column("elapsed_time", BIGINT, "Elapsed time in seconds this process has been running."),
```

Comment:
How was this value different than `start_time`?

Reply:
Start time would be a time that does not change. elapsed time would increment every second that the process is alive